modules/ROOT/pages/_partials/include-insights-agentic-details.adoc

```diff
@@ -0,0 +1,74 @@
+
+//CHARTS/INFOGRAPHICS: AGENTS
+//tag:charts[]
+//end:charts[]
+
+//METADATA ON DETAILS PAGES (SOURCE: API MANAGER)
+//tag::metadata[]
+
+//IN MOCK
+//  TODO: Is this unique even if there are multiple instances of the same API? Confusing!
+* *API Instance ID*: 
+Unique identifier that API Manager generates for the instance 
+
+// IN MOCK
+//  TODO: the config is optional
+* *API Label*:
+Unique identifier configured in API Manager as an optional instance property that distinguishes one instance from another instance of the same API
+
+//IN MOCK
+* *API Status*:
+*Active*, *Inactive*, or *Unregistered* status for the instance
++
+For more detail, see xref:api-manager*:latest-overview-concept.adoc#tracking-registration-status[Tracking Registration Status] in the API Manager documentation.
++
+Applies to agents, brokers, and MCP servers
+
+//  IN MOCK
+//  TODO: VERIFY DESCRIPTION
+* *API Version*: 
+Version of the API (example: *V1*) that the agent, broker, or MCP server instantiates
+
+//  IN MOCK
+//  TODO: VERIFY DESCRIPTION
+* *Asset Version*:
+Version of the API displayed in Anypoint Exchange that the instance implements
+Examples: *1.0.0 (Latest)*, *1.0.2*
+
+//  IN MOCK
+//  TODO: VERIFY DESCRIPTION -- this config is optional 
+* *Consumer Endpoint*:
+If configured, provides a proxy application URI that consumers use when making requests to the instance
+
+//  IN MOCK
+* *Implementation URI*:
+Upstream URI on the backend
+
+//TODO for the instance
+
+* *Instance Conformance*:
+Most severe conformance issue level detected by API Governance
+//TODO: POINT TO LIST OF VALUES
```

Comment:
@sathishpaulleo Do you know how I can get a list of the accepted conformance values?

```suggestion
* *Instance Conformance*:
Most severe conformance issue level detected by API Governance
//TODO: POINT TO LIST OF VALUES
```